server/controllers/profile.controller.js

```diff
@@ -131,6 +136,22 @@ class ProfileController {
     };
     return Response(res, 200, 'Successfully unfollowed user', user);
   }
+
+  static async getNotifications(req, res) {
+    const { user: { id } } = req;
+
+    const notifications = await Notification.findAndCountAll({
+      where: {
+        userId: id,
+        isSeen: false
+      },
+      order: [['createdAt', 'DESC']]
+    });
+    const { count, rows } = notifications;
+    if (!rows.length) return Response(res, 200, 'No new notifications.', []);
```

Comment:
Nice job @TheDrizzyWay. Since we are returning `200` for when notifications are present and not I think we can just return it once. Leaving the frontend to check for `rows.length`. What do you think?

Reply:
On point. Fixing that now